backend/src/main/java/com/festago/domain/Stage.java

```diff
@@ -60,6 +60,10 @@ private void validate(LocalDateTime startTime, LocalDateTime ticketOpenTime, Fes
         }
     }
 
+    public boolean isStart(LocalDateTime time) {
+        return startTime.isBefore(time);
+    }
```

Comment:
```suggestion
    public boolean isStart(LocalDateTime currentTime) {
        return currentTime.isAfter(startTime);
    }
```
이게 조금 더 가독성이 좋은 것 같아요! 현재시간이 무대시작시간 이후이다!
(currentTime이라고 변수명 지은 이유는 다른 도메인 메서드 파라미터로 그렇게 사용한 것 같아서입니다!)